WrittenKitten.js

```diff
@@ -160,18 +158,17 @@ function set_search(searchTerm) {
 	if (tmp = getParameterByName("search")) {
 		tmp.replace(/</g, "&lt;").replace(/>/g, "&gt;"); // sanitize
 		search_for = tmp;
-		flickr_search_term = tmp + ',cute';
 	} else {
-		search_for = searchTerm + ',cute';
```

Comment:
Don't need to add 'cute' multiple times -- the code already adds it at the start of every search, and not adding it here, too, saves some labor when setting the title.